03/js-tools-note.md

```diff
@@ -0,0 +1,10 @@
+## What I’ve learned
+- Reviewed my knowledge of JS tools
+- Discovered some new and interesting tools I hadn’t used before
```

Comment:
Jakie to te narzędzia? Za miesiąc już nie będziesz pamiętał - warto pisać konkrety ;)